nginx.conf

```diff
@@ -36,6 +36,10 @@
 
     client_max_body_size 10m;
 
+    # This root stores static content for general pmm-server purposes.
+    # Ex.: local-rss.xml - contains Percona's news when no internet connection.
+    root /usr/share/pmm-server/static;
```

Comment:
we can use alias for this to avoid affecting other requests.

Reply:
here https://github.com/percona/pmm-server/pull/217/files#diff-daaf2fceca49ed034eace91c4e875653R188

Reply:
no, the alias does not here.